src/data/polygons_2019.json

```diff
@@ -488,32 +604,44 @@
         "show_icon": "",
         "icon": "",
         "rotate": "",
-        "left_panel": ""
+        "left_panel": true
```

Comment:
What’s going on here?

Reply:
After making the polygon geometry edits, including the addition of Chophouse Row, I copied over the polygons from the combined JSON into the polygon-only JSON for consistency and to pull in the labels for Chophouse Row since the polygon json is only being used for labeling. A side effect of this is that the changes from [your PR](https://github.com/spatialdev/react-chbp/commit/8a97ff0080bc282ac1efbab13fb1523eba19e71e#diff-d7714683cc599b32858d6b7f5759b95a) from earlier that added some features to the left panel are carried over into the polygons-only json. If this causes any display issues, I will pull these back out.